### PR TITLE
Added tests for Tokens class

### PR DESCRIPTION
**Pull request by TechFreak**

### CODE REVIEW

File: src/main/java/com/authy/api/Tokens.java [hunk 1]
```diff
@@ -26,29 +29,24 @@ public Tokens(String uri, String key, boolean testFlag) {
         super(uri, key, testFlag);
     }
 
-    public Token verify(int userId, String token) {
+    public Token verify(int userId, String token) throws UnsupportedEncodingException, AuthyException {
         return verify(userId, token, null);
     }
 
-    public Token verify(int userId, String token, Map<String, String> options) {
+    public Token verify(int userId, String token, Map<String, String> options) throws AuthyException, UnsupportedEncodingException {
```
Comment (yuniersoad) [inline src/main/java/com/authy/api/Tokens.java]: Not sure if we should change the method signature, that will break existing clients of the lib.

About the `UnsupportedEncodingException` IMO we don't really need the `URLEncoder.encode` that produces it, neither `token` nor `userId ` should contain characters that requiere special encoding.

Comment (TechFreak) [reply src/main/java/com/authy/api/Tokens.java]: Agree on the UnsupportedEncodingException, I removed the code that caused that exception to be thrown. But I think not having the AuthyException on the signature was actually a bug, we were failing silently and not giving any signal to the user that an error had happened

File: src/test/java/com/authy/api/TokensTest.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
+package com.authy.api;
```
Comment (yuniersoad) [inline src/test/java/com/authy/api/TokensTest.java]: Some of the tests should `verify` the HTTP request received by the mock server.

Comment (TechFreak) [reply src/test/java/com/authy/api/TokensTest.java]: done

**Pull request merged**